src/app/components/todos.component/todos.component.html

```diff
@@ -0,0 +1,18 @@
+<h3>Todo list</h3>
+<form novalidate (ngSubmit)="submitTodo($event)">
+  <input type="text" [(ngModel)]="textTodo" name="todo" placeholder="Your text">
+  <button type="submit">Send</button>
+</form>
+
+<ul *ngIf="todos.length">
+  <li *ngFor="let todo of todos">
+    <label>
```

Comment:
It is better to extract TodoComponent out of this. Whenever you have ngFor in your template - consider to extract its body as a separate component